src/Service/TwitterService.php

```diff
@@ -36,4 +33,23 @@ public static function get($request, $params = []) {
 
         return $result;
     }
+
+    public function getTweetsSince($last_id) {
+        return $this->get('https://api.twitter.com/1.1/statuses/user_timeline.json', [
+            'screen_name' => $this->screen_name,
+            'since_id' => $last_id
+        ]);
+    }
+
+    public function getTweet($id) {
+        return $this->get('https://api.twitter.com/1.1/statuses/show.json', [
+            'id' => $id
+        ]);
+    }
+
+    public function getTweetURL($tweet) {
+        $screen_name = $tweet->user->screen_name;
+        $id_str = $tweet->id_str;
+        return "https://twitter.com/$screen_name/status/$id_str";
```

Comment:
You could also use `sprintf` to interpolate the strings. Matter of personal preference.